packages/front-end/services/experiments.ts

```diff
@@ -56,6 +56,31 @@ export function isBelowMinChange(
   return Math.abs(baseline.cr - stats.cr) / baseline.cr < minPercentChange;
 }
 
+export function shouldHighlight({
```

Comment:
Thanks, I know this is kind of annoying, but having the business logic in as few places as possible has always served me well.